website/docs/d/pi_instance.html.markdown

```diff
@@ -70,3 +70,4 @@ In addition to all argument reference list, you can access the following attribu
 - `storage_type` - (String) Storage type where server is deployed.
 - `virtual_cores_assigned` - (Integer) The virtual cores that are assigned to the instance.
 - `volumes`- (List of strings) The list of volume IDs that are attached to the instance.
+- `placement_group_id`- (String) The ID of the placement group that the instance is in if any.
```

Comment:
Need to sort the attribute names alphabetically in all the doc changes.

Reply:
Done.